### DISCUSSION TITLE
Typo and Suggestion (Teleport)

### DISCUSSION THREAD

**Discussion by ccrim** [flightsimcloud/advanced-vfr-map#8] [Ideas]
Hey, this is a really nice tool.  Thanks for putting this together!

Two things: 1) note that "planner" is spelled with two n's in English (not "planer")
and 2) would it be possible to integrate a "teleport" feature to instantly move your plane to another location (and elevation) on the planet?  This is currently supported via the tool "vfrmap" but it doesn't provide the in-game map display and multiple map sources that you do.  With vfrmap, you have to tab out to your web browser (or use an alternate device) to look at the map or perform the teleport.

Thanks!

**Comment by toorop** [comment]
use the flight planer, set a point where you want to be teleported and click on teleport icon (the cloud) ;)
